talentsearch/resources/js/components/Dashboard.tsx

```diff
@@ -1,36 +1,13 @@
 import React, { ReactElement } from "react";
```

Comment:
Not sure what else this should be called, `Container.tsx`? I'm open to suggestions.

Reply:
What about Home.tsx

Reply:
I like your line of thought with Home but I think that's not quite right, since this component will be used on _all_ the talentsearch pages. Container makes sense, but is a bit too generic. Maybe Page or PageContainer? 

And SearchDashboard could be renamed to something like Root, or Router.